app.js

```diff
@@ -1,14 +1,43 @@
-import express from 'express';
+const express = require("express");
 const app = express();
+const regeneratorRuntime = require("regenerator-runtime");
 
 const environment = process.env.NODE_ENV || 'development'; 
 const configuration = require('./knexfile')[environment];
 const database = require('knex')(configuration);
 
 app.use(express.json());
 
-app.get('/', (request, response) => {
-  response.send('Looks like the server is running!');
+app.get("/", (request, response) => {
+  response.send("We're going to test all the routes!");
```

Comment:
Should we remove this root path and the associated test from our finished file?

Reply:
I think we should leave them to show full test coverage, but agree they aren't pulling much weight.